beekeeper-core/src/test/java/com/expediagroup/beekeeper/core/hive/HiveClientTest.java

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.core.hive;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.metastore.api.NoSuchObjectException;
+import org.apache.hadoop.hive.metastore.api.UnknownDBException;
+import org.apache.thrift.TException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+import com.expediagroup.beekeeper.core.error.BeekeeperException;
+
+import com.hotels.hcommon.hive.metastore.client.api.CloseableMetaStoreClient;
+
+public class HiveClientTest {
+
+  private @Mock CloseableMetaStoreClient client = Mockito.mock(CloseableMetaStoreClient.class);
```

Comment:
I don't think you need to create the mock object after you use the mock annotation
```suggestion
  @Mock 
  private CloseableMetaStoreClient client;
```

Reply:
It actually doesnt work if you dont create the mock object, but I can remove the annotation 👍